staff/almu-itur/cv-styled/styles/styles.css

```diff
@@ -0,0 +1,282 @@
+* {
+    font-family: 'Raleway', sans-serif;
+}
+
+@media screen and (min-device-width: 481px) and (max-device-width: 768px) and (orientation:portrait) {
+    
+        header {
+            border: 0.5em solid grey;
+            padding: 1.25em;
+        }
+
+        h1 {
+            color: back;
+            margin-left: 0.7em;
+        }
+
+        h2 {
+            color: grey;
+            margin-left: 0.7em;
+        } 
+
+        h3 {
+            margin-left: 0.7em;
+        }
+
+        h4 {
+            margin-left: 0.7em;
+        }
+
+        h5 {
+            margin-left: 0.7em;
+        }
+
+        p {
+            margin-left: 0.7em;
+        }
+
+        a {
+            margin-left: 0.7em;
+        }
+
+        .container {
+            margin-top: 1.25em;
+            display: grid;
+            grid-row: repeat (8, 1fr);
+            grid-row-gap: 1.25em;
+        }
+        .summary {
+            grid-row: 1 / span 1;
+            background-color: #d9dddd;
+        }
+    
+        .experience {
+            grid-row: 2 / span 2;
+            background-color: #f3f6f6;
+        }
+    
+        .education {
+            grid-row: 4 / span 1;
+            background-color: #d9dddd;
+        }
+    
+        .itskills {
+            grid-row: 5 / span 1;
+            background-color:#a2dfcc;
+        }
+    
+        .languages {
+            grid-row: 6 / span 1;
+            background-color: #a1e1df;
+    
+        }
+    
+        .courses {
+            grid-row: 7 / span 1;
+            background-color:#9adfea;
+        }
+    
+        .transversal {
+            grid-row: 8 / span 1;
+            background-color: #9ab3ea;
+        }
+    
+        footer {
+            margin: 1.25em / 0;
+            background-color: #33b1c3;
+            height: 3em;
+        } 
+
+}
+
+
+@media screen and (min-device-width: 481px) and (max-device-width: 768px) and (orientation:landscape) {
+    
+
+        header {
+            border: 0.5em solid grey;
+            padding: 1.25em;
+        }
+    
+        h1 {
+            color: back;
+            margin-left: 0.7em;
+        }
+    
+        h2 {
+            color: grey;
+            margin-left: 0.7em;
+        } 
+    
+        h3 {
+            margin-left: 0.7em;
+        }
+    
+        h4 {
+            margin-left: 0.7em;
+        }
+    
+        h5 {
+            margin-left: 0.7em;
+        }
+    
+        p {
+            margin-left: 0.7em;
+        }
+    
+        a {
+            margin-left: 0.7em;
+        }    
+    
+        .container {
+            margin-top: 1.25em;
+            display: grid;
+            grid-template: 1fr 1fr 1fr 1fr / 3fr 1fr;
```

Comment:
try using repeat (4, 1fr) instead of typing each column individually

Reply:
Done, thanks!